modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
```

Comment:
Consider adding a _Prerequisites_ heading for this _Procedure_ because the procedure assumes that the user has access to OpenShift CLI (`oc`) and Infoblox UI.